### DISCUSSION TITLE
VITS repo says last commit is "3 years ago"...

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [rhasspy/piper#666] [General]
Hello there!

AI and inference solutions have come quite a way in but a single year. Same goes for the underlying tech stack in use. Chances are, that with VITS being kinda "old", there ought to be a newer, potentially faster method around.

I am looking into putting together a voice myself, but I only have one 4090 right now. Hence why I am looking at the various methods of building the voice model and running inference with it.

Do you have any pointers, by chance?

Thank you and kind regards,
Ingwie

**Comment by synesthesiam** [comment]
I've heard good things about F5-TTS: https://github.com/SWivid/F5-TTS/

My goal with Piper was never to chase the latest and greatest models, but instead to be "boring" and reliable :smile: 
There are definitely improvements that need to be made to this repo, but I see no reason to move on from VITS just because it's "old".

**Reply by xsploit** [reply]
piper is great but it just lacks custom models i wish we could convert vits .pth to onnx and it work

**Reply by simpleC0de** [reply]
> piper is great but it just lacks custom models i wish we could convert vits .pth to onnx and it work

you can train your own model

**Reply by RobertAgee** [reply]
> piper is great but it just lacks custom models i wish we could convert vits .pth to onnx and it work

Look up [TextyMcSpeechy](https://github.com/domesticatedviking/TextyMcSpeechy/), basically you can use a voice cloner to create higher quality datasets and then finetune on a checkpoint in piper to get a inference model.

**Reply by xsploit** [reply]
> > piper is great but it just lacks custom models i wish we could convert vits .pth to onnx and it work
> 
> Look up [TextyMcSpeechy](https://github.com/domesticatedviking/TextyMcSpeechy/), basically you can use a voice cloner to create higher quality datasets and then finetune on a checkpoint in piper to get a inference model.

Im aware but its a lot of work i dont want my 4060 to go burr for 8 hrs plus lol. Point is other tts like rvc and gpt sovits have models or all types to download.  Gpt sovits is more taxing but ive been able to render in chunks for a more realtime experience

**Reply by RobertAgee** [reply]
Google colab is free!!~~ But I feel ya, I wouldn't want to run it on my own GPU either. I think the distinct advantage over other TTS solutions is the quality/compute ratio and that it can run on smaller devices, if needed. For me, I want to embed it in chat applications, so I understand the desire for custom voices.

**Reply by xsploit** [reply]
Yea if i do train another model it was be with a service. Right now i got a ai waifu ....so i need a anime voice for piper.
